coredump_bot.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=CoredumpBot
```

Comment:
If you want to make sure that it isn't launched before network is up, add `After=network.target`.